File: 1 - Introduction & stratégies de recherche/Problèmes/I - Equation/A.py
def is_composite(n):
	c = 0;
	for i in range(1, n+1):
		if n % i == 0:
			c+=1
	if c > 2:
		return True
	return False

n = int(input())
a = 2
b = a+n

for i in range(0, 1000000001):
	if is_composite(a) and is_composite(b):
		print(b, " ", a)
		break
	a+=1;
	b+=1;

File: 1 - Introduction & stratégies de recherche/Problèmes/I - Equation/A.c
#include <stdio.h>


int is_composite(int n){

	int i;
	int c = 0;
	for(i=1; i<=n; i++)
	{
		if(n % i == 0)
			c++;
	}
	if(c > 2)
		return 1;
	return 0;

}

int main(void) {
	
	int n;
 	scanf("%d", &n);
 	
 	int a = 2;
 	int b = a+n;
 	long i;
 	for(i = 0; i <= 1000000000; ++i){
 		
 		if (is_composite(a) && is_composite(b)) {
 			printf("%d %d\n", b, a);
 			break;
 		}
 		a++;
 		b++;
 	
 	}
 	
 	
	return 0;
}


